ui/app/models/config-ui/message.js

```diff
@@ -5,18 +5,50 @@
 
 import Model, { attr } from '@ember-data/model';
 import lazyCapabilities, { apiPath } from 'vault/macros/lazy-capabilities';
-import { isAfter } from 'date-fns';
+import { format, addDays, startOfDay, isAfter } from 'date-fns';
 import { parseAPITimestamp } from 'core/utils/date-formatters';
+import { withModelValidations } from 'vault/decorators/model-validations';
+import { withFormFields } from 'vault/decorators/model-form-fields';
 
+export const localDateTimeString = "yyyy-MM-dd'T'HH:mm";
+
+const validations = {
+  title: [{ type: 'presence', message: 'Title is required.' }],
+  message: [{ type: 'presence', message: 'Message is required.' }],
+};
+
+@withFormFields(['authenticated', 'type', 'title', 'message', 'linkTitle', 'startTime', 'endTime'])
+@withModelValidations(validations)
```

Comment:
Follow up task: `defaultValue` will be removed for type and authenticated once the backend is updated.